### ISSUE TITLE
static-html package error

### ISSUE THREAD

**Issue by SPAHI4** [meteor/meteor#5147]
```
meteor add static-html
 => Errors while adding packages:

While selecting package versions:
error: unknown package: isobuild:compiler-plugin
Required by: ecmascript 0.1.3-plugins.5
Required by: static-html 0.0.2-plugins.0

unknown package: isobuild:isopack-2
Required by: caching-compiler 1.0.0-plugins.1
Required by: templating-tools 0.0.2-plugins.0
Required by: caching-html-compiler 1.0.1-plugins.0
```

**Comment by SPAHI4** [comment]
i tried to do this
```
meteor --release PLUGINS-PREVIEW@2 add ecmascript
```
and get this error
```
meteor add static-html
 => Errors while adding packages:

While selecting package versions:
error: unknown package: isobuild:compiler-plugin
Required by: ecmascript 0.1.3-plugins.5
Required by: static-html 0.0.2-plugins.0

unknown package: isobuild:isopack-2
Required by: caching-compiler 1.0.0-plugins.1
Required by: templating-tools 0.0.2-plugins.0
Required by: caching-html-compiler 1.0.1-plugins.0

```

**Comment by SPAHI4** [comment]
@stubailo so what i must do now? Wait for release?

**Comment by stubailo** [comment]
Are you running one of the RCs for Meteor 1.2? Or 1.1.0.3?

**Comment by SPAHI4** [comment]
No, 1.1.0.3 on windows

**Comment by stubailo** [comment]
Yeah, this package will only be available in 1.2, but you can try it out now by using a release candidate with `meteor update --release 1.2-rc.14`. Unfortunately it's hard to have a better error for this.
